storage/persistent_storage/persistent-storage-azure-file.adoc

```diff
@@ -27,6 +27,15 @@ storage provider.
 Azure File volumes use Server Message Block.
 ====
 
+[IMPORTANT]
+====
+{product-title} defaults to using an in-tree (non-CSI) plug-in to provision Azure File storage.
```

Comment:
We should remove this line because by default OCP doesn't create any StorageClass for Azure File,